tests/test_diag_table.py

```diff
@@ -28,12 +28,12 @@ def diag_table():
 
 def test_DiagTable_string_round_trip(diag_table):
     round_tripped_diag_table = DiagTable.from_str(str(diag_table))
-    assert str(diag_table) == str(round_tripped_diag_table)
+    assert diag_table == round_tripped_diag_table
```

Comment:
now that `DiagTable` is a dataclass and has the `__eq__` method, don't need to use `str` for equality checking.